tensilelite/Tensile/SolutionStructs/Solution.py

```diff
@@ -1115,6 +1115,7 @@ def assignDerivedParameters(
         if state["LocalSplitU"] > 1:
           reject(state, printRejectionReason, "Atomic Stream-K not working with LocalSplitU")
       if not state["Valid"]:
+        print2("in assignDerivedParameters, state['Valid'] = False")
```

Comment:
Please add the following assert such as
if StreamK == False, GSU must > 0
if StreamK == True, GSU must == 0

Reply:
Fixed